File: Object Oriented Techniques using Java Lab/LAB 02/Factorial.java
package q10886;
class Factorial
{
	public static void main(String args[])
	{
	int n;
	int fact = 1;
	n = Integer.parseInt(args[0]);
	for(int i=1;i<=n;i++)
	{
	fact=fact*i;
	}
	System.out.println("Factorial of "+n+" is "+fact);
	}
}

File: Data Structures Lab using Python/5. Recursion/factorial.py
#factorial using tail recursion
def factorial(num):
	if num==0:
		return 1   #base_case
	else:
		return num*factorial(num-1)   #recursive_call
value=int(input("Enter a number: "))
fact=factorial(value)
print("The factorial of",value,"is",fact)
